setup.ts

```diff
@@ -48,6 +49,39 @@ export async function main(): Promise<void> {
     });
   }
 
+  let shouldSetRemoteHost: boolean;
+
+  if (process.env.REACT_APP_REMOTE_HOST) {
+    console.log(
+      `\nEndpoint for the remote url already exists with the value:\n${process.env.REACT_APP_REMOTE_HOST}`,
+    );
+    shouldSetRemoteHost = true;
+  } else {
+    const { shouldSetRemoteHostResponse } = await inquirer.prompt({
+      type: 'confirm',
+      name: 'shouldSetRemoteHostResponse',
+      message: 'Would you like to set up a remote host?',
+      default: true,
+    });
+    shouldSetRemoteHost = shouldSetRemoteHostResponse;
```

Comment:
1. This is unnecessary, setup already asks for the `REACT_APP_TALAWA_URL`  value that includes a TCP port. BTW, You cannot assume the port will never change. 
2. You are duplicating the prompting for an existing parameter. You should be able to derive the necessary information from the `REACT_APP_TALAWA_URL`  value.

Reply:
This is not asking for API endpoint, but for solving the problem.
If we are accessing the admin portal from the network and the API endpoint contains "localhost", it causes the error. So asking for the remote URL is to change the URL into a URL that can be accessed without any problem.